docusaurus/docs/introduction/backend.md

```diff
@@ -83,18 +79,30 @@ Examples of use cases for implementing resources:
 - Extend Grafana's HTTP API with custom resources, methods and actions.
 - Use [chunked transfer encoding](https://en.wikipedia.org/wiki/Chunked_transfer_encoding) to return large data responses in chunks or to enable certain streaming capabilities.
 
-#### Health checks
+### Health checks
 
 The health checks capability allows a backend plugin to return the status of the plugin. For data source backend plugins, the health check is automatically called when a user edits a data source and selects _Save & Test_ in the UI.
 
 A plugin's health check endpoint is exposed in the Grafana HTTP API and allows external systems to continuously poll the plugin's health to make sure that it's running and working as expected.
 
-#### Collect metrics
+### Collect metrics
 
 A backend plugin can collect and return runtime, process, and custom metrics using the text-based Prometheus [exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/). If you're using the [Grafana Plugin SDK for Go](./grafana-plugin-sdk-for-go.md) to implement your backend plugin, then the [Prometheus instrumentation library for Go applications](https://github.com/prometheus/client_golang) is built-in. This SDK gives you Go runtime metrics and process metrics out of the box. You can use the [Prometheus instrumentation library](https://github.com/prometheus/client_golang) to add custom metrics to instrument your backend plugin.
 
 The Grafana HTTP API offers an endpoint (`/api/plugins/<plugin id>/metrics`) that allows you to configure a Prometheus instance to scrape the metrics.
 
-#### Streaming
+### Streaming
+
+The streaming capability allows a backend plugin to handle data source queries that are streaming. For more information, refer to [Build a streaming data source plugin](https://grafana.com/docs/grafana/latest/developers/plugins/create-a-grafana-plugin/develop-a-plugin/build-a-streaming-data-source-plugin/).
+
+## Data communication model
+
+When communicating with backend plugins, Grafana should provide all the necessary information (configuration) in each request to allow the plugin to fulfill the request and return a response. This is a simple model for ensuring data communication between Grafana and plugin. 
```

Comment:
This might be simplified. Something along these lines. Not sure the last phrase even is needed.

```suggestion
Grafana provides all the necessary information (configuration) in each request to backend plugin, allowing the plugin to fulfill the request and return a response. This simplified communication model makes it easy to reason about removes burden from plugin authors. 
```

Reply:
I think in "makes it easy to reason about removes burden..." there are two thoughts that didn't get melded together the right way.

Reply:
I kind of agree that:
> Not sure the last phrase even is needed.

How about:
> Grafana uses a simplified model where all necessary information (configuration) is provided in each request to a backend plugin, allowing the plugin to fulfil the request and return a response.